doc/howto/instances_manage.md

```diff
@@ -76,3 +76,29 @@ There are two ways to prevent accidental deletion of instances:
 
 - To protect a specific instance from being deleted, set [`security.protection.delete`](instance-options-security) to `true` for the instance.
   See {ref}`instances-configure` for instructions.
+
+### Rebuild an instance
+
+If you want to keep the same instance configuration, but you want to freshly start with a new instance content, you have the option to rebuild it.
+
+Once you have stopped your instance and that you ensured that the instance has no snapshots, you can proceed with the rebuild operation. It can be done in three different ways:
+
+- Rebuild the instance with its original image (the one used to create the instance).
+- Rebuild the instance with a new image.
+- Rebuild the instance as empty.
+
+```
+user@workstation:~$ lxc list
++---------+---------+-----------------------+-----------------------------------------------+-----------+-----------+
+|  NAME   |  STATE  |         IPV4          |                     IPV6                      |   TYPE    | SNAPSHOTS |
++---------+---------+-----------------------+-----------------------------------------------+-----------+-----------+
+| c1      | RUNNING | 10.131.188.152 (eth0) | fd42:cf18:4dc3:87f7:216:3eff:fe83:dd60 (eth0) | CONTAINER | 0         |
++---------+---------+-----------------------+-----------------------------------------------+-----------+-----------+
+
+user@workstation:~$ lxc stop c1                            # We need to stop the instance before rebuilding it.
```

Comment:
Can you follow the existing command examples, I don't think we include the user@workstation bit, for example.